instrumentation/play/play-mvc/play-mvc-2.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/play/v2_4/PlayInstrumentationModule.java

```diff
@@ -17,8 +17,11 @@
 @AutoService(InstrumentationModule.class)
 public class PlayInstrumentationModule extends InstrumentationModule {
 
+  static final String INSTRUMENTATION_NAME = "play-mvc";
+  static final String INSTRUMENTATION_NAME_EXTRA = "play-mvc-2.4";
+
   public PlayInstrumentationModule() {
-    super("play", "play-2.4");
+    super(INSTRUMENTATION_NAME, INSTRUMENTATION_NAME_EXTRA);
```

Comment:
I think good to stay consistent with the other instrumentations, and propose something like this as a separate change that if agreed on we apply everywhere

Reply:
Meh, ok. I'll change it back. 🙃 
I only wanted a constant there because it would have been easy to overlook the corresponding change in the singletons class and have a mismatch. But it's fine, it's only two places.

Reply:
ya, we have this problem everywhere (and I agree it's a problem and easy to miss), the other option we've discussed is adding a test to verify that module name == instrumentation name == instrumentation scope name